src/commands/infra/validate.decorator.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
Though the decorator is written here, on my machine I see 2 generate references in the help menu for `spk infra`

```
naterose@naterose-Blade:~/Desktop/Microsoft/fy20/nate_fork/spk/dist$ ./spk-linux infra 
Usage: infra [options] [command]

Manage and modify your Bedrock infrastructure.

Options:
  -v, --verbose         Enable verbose logging
  -h, --help            output usage information

Commands:
  generate|g [options]  Generate scaffold for terraform cluster deployment.
  scaffold|s [options]  Create initial scaffolding for cluster deployment.
  generate|g [options]  Generate scaffold for terraform cluster deployment.
naterose@naterose-Blade:~/Desktop/Microsoft/fy20/nate_fork/spk/dist$ git branch
* (HEAD detached at upstream/refactorValidateTs)
```

Reply:
sorry, it is a bug in my implementation.

Reply:
i am getting this, after the fix
```
Usage: infra [options] [command]

Manage and modify your Bedrock infrastructure.

Options:
  -v, --verbose         Enable verbose logging
  -h, --help            output usage information

Commands:
  generate|g [options]  Generate scaffold for terraform cluster deployment.
  scaffold|s [options]  Create initial scaffolding for cluster deployment.
  validate|v            Validate will verify that all infrastructure deployment prerequisites have been correctly installed.
```